public/app/features/correlations/CorrelationsPage.tsx

```diff
@@ -157,7 +158,9 @@ export default function CorrelationsPage() {
       navModel={navModel}
       subTitle={
         <>
-          Define how data living in different data sources relates to each other. Read more in the{' '}
+          <Trans i18nKey="correlations.sub-title">
```

Comment:
Links should probably be part of the translations to allow linking to docs with a specific locale. Also, in this case the link label is not translated:

<img width="1263" alt="Screenshot 2023-09-19 at 12 17 14" src="https://github.com/grafana/grafana/assets/745532/bd59da78-3b44-4aa5-a6e7-ffccbd1f42be">

Reply:
(FYI, we don't support this. There's no need to translate docs links)